packages/frontend/src/config/providers.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { ReactQueryDevtools } from "react-query/devtools";
+
+const queryClient = new QueryClient();
+
+export const Providers: React.FC = ({ children }) => (
```

Comment:
nazwałbym to QueryProvider bo providers mi nic nie mówi na temat tego co to jest